packages/client/wallets/aa/src/blockchain/wallets/passkey.ts

```diff
@@ -103,4 +105,10 @@ export class PasskeyWalletService {
             type: "passkeys",
         };
     }
+
+    private setJwtCookie(jwt: string) {
+        const now = new Date();
+        const expireTime = new Date(now.getTime() + 1 * 60 * 60 * 1000); // Now + 1 hour
+        document.cookie = `user_jwt=${jwt}; expires=${expireTime.toUTCString()}; path=/`;
```

Comment:
If you don't specify it, this cookie will only be sent to servers w/ the domain of the client that set it. When `harrypotter.com` uses SDK, `crossmint.com` won't receive the cookies.

Reply:
I read a bit on this, what options do we have? We can't set cookies cross domains. Should we set the cookie server side (for crossmint.com) on the passkeys server on let's say the first interaction?

Reply:
That sounds like a good idea to me, maybe we could have some "handshake" endpoint on the passkeys controller to set the cookie